unlock-app/src/pages/paywall.js

```diff
@@ -6,11 +6,15 @@ import DeveloperOverlay from '../components/developer/DeveloperOverlay'
 import ShowUnlessUserHasKeyToAnyLock from '../components/lock/ShowUnlessUserHasKeyToAnyLock'
 import { LOCK_PATH_NAME_REGEXP } from '../constants'
 import BrowserOnly from '../components/helpers/BrowserOnly'
+import UnlockedFlag from '../components/lock/UnlockedFlag'
 
 const Paywall = ({ lock }) => {
   return (
     <BrowserOnly>
-      <ShowUnlessUserHasKeyToAnyLock locks={lock ? [lock] : []}>
+      <ShowUnlessUserHasKeyToAnyLock
+        locks={lock ? [lock] : []}
+        else={<UnlockedFlag locks={lock ? [lock] : []} />}
```

Comment:
I am not sure I like that syntax very much. Why is the `then` a children and the `else` an attribute?
I think this re-inforces the need for `ShowUnlessUserHasKeyToAnyLock` to be a helper function

Reply:
could we do this as a subsequent pull request, open an issue to remember it?

Reply:
also, brief note, this is how `Suspense` works. the "else" is called "fallback" but this is a pretty standard pattern.